src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
+      <div class="calendar calendar--month-length-30 calendar--start-day-sun">
```

Comment:
```suggestion
      <div class="calendar calendar--30 calendar--sun">
```
Можно вот так оставить